src/main/java/com/example/bookshop/dto/CreateBookRequestDto.java

```diff
@@ -1,14 +1,23 @@
 package com.example.bookshop.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
     private String isbn;
+    @NotNull
+    @Min(0)
     private BigDecimal price;
+    @NotNull
     private String description;
+    @NotNull
     private String coverImage;
```

Comment:
Please make validations more useful.
Check where reasonable to apply `@NotNull`, `@NotBlank`, `@Length`, `@Isbn` and other annotations